src/DynamoCoreWpf/ViewModels/Core/AnnotationViewModel.cs

```diff
@@ -0,0 +1,187 @@
+﻿using System.Collections.Generic;
+using System.Configuration;
+using System.Data;
+using System.Drawing;
+using System.Windows;
+using System.Windows.Controls;
+using System.Windows.Documents;
+using System.Windows.Media;
+using System.Windows.Media.Animation;
+using Dynamo.Models;
+using System;
+using Dynamo.UI.Commands;
+using Color = System.Windows.Media.Color;
+
+namespace Dynamo.ViewModels
+{
+    public class AnnotationViewModel : ViewModelBase
+    {
+        private AnnotationModel _annotationModel;
```

Comment:
Again, do not use _ with private fields.  Just do lowercase: http://stackoverflow.com/a/450418/984515